README.md

```diff
@@ -50,11 +63,23 @@ To install the MongoDB Community Kubernetes Operator:
       kubectl get pods --namespace <my-namespace>
       ```
 
-## Usage
+
+## Upgrade the Operator
+
+To upgrade the MongoDB Community Kubernetes Operator:
+
+1. Change to the directory in which you cloned the repository.
+2. Invoke the following `kubectl` command to upgrade the [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
+   ```
+   kubectl apply -f deploy/crds/mongodb.com_mongodb_crd.yaml --namespace <my-namespace>
```

Comment:
```suggestion
   kubectl apply -f deploy/crds/mongodb.com_mongodb_crd.yaml
```

crds are cluster wide resources, so we don't need to include the `--namespace`

Reply:
Whoops, good catch! Fixed.